scripts/raceday_scheduler.py

```
@@ -0,0 +1,195 @@
+
+    print(f"$ {' '.join(cmd)}\n", flush=True)
+    result = subprocess.run(cmd, cwd=str(PROJECT_ROOT))
+    return result.returncode
+
+
+def print_schedule(from_hhmm: str | None, dry_run: bool):
+    today = date.today()
+    print(f"{'='*65}", flush=True)
+    print(f"  Race Day Scheduler  —  {today} (JRA Saturday)", flush=True)
+    if dry_run:
+        print(f"  Mode : DRY RUN", flush=True)
```

Ruff (0.15.7)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)